### PR TITLE
Accessibility Changes

### PR DESCRIPTION
**Pull request by Bad-At-Usernames**
I've fixed a few issues from #50, as well as some problems with icons.

### CODE REVIEW

**Review by Geometrically — APPROVED** [review]
LGTM. Will wait for false to take a look before merged. Thanks for the contribution!

**Review by falseresync — APPROVED** [review]

**Pull request merged**
